test/utils.jl

```diff
@@ -41,7 +41,7 @@ N = GenericArray([1.0 0.0; 1.0 0.0])
 n = 10
 areal = randn(n,n)/2
 aimg  = randn(n,n)/2
-@testset "For A containing $eltya" for eltya in (Float32, Float64, ComplexF32, ComplexF64, Int)
+@testset "For A containing $eltya" for eltya in (Float32, AbstractFloat, ComplexF32, ComplexF64, Int)
```

Comment:
These changes are much too blunt, especially in the test files. Here (^) for example, where the point is to loop through different concrete `eltype`s.

Reply:
Probably the tests should have been excluded from the regex.